lib/config.js

```diff
@@ -18,15 +18,18 @@ config.express.session.secret = '0123456789abcdef';
 config.express.session.key = 'bedrock.sid';
 config.express.session.prefix = 'bedrock.';
 config.express.session.resave = true;
-config.express.session.saveUninitialized = true;
+// sessions that have not been modified are not saved to the db
+// until modified.
+config.express.session.saveUninitialized = false;
 config.express.session.cookie = {};
 config.express.session.cookie.secure = true;
-// NOTE: 'connect' doesn't update the expires age for the cookie on every
-//   request so sessions will always timeout on the client after the maxAge
-//   time. Setting to null will cause sessions checks to only happen on the
-//   server which does update the expires time on every request. The server
-//   session defaultExpirationTime is set below.
+// NOTE: setting maxAge will allow cookies to persist
+// after the ttl has expired
```

Comment:
This note seems less informative than the one that is being deleted. If can we keep the old note too? If there's something incorrect with it, can it be adjusted?

Reply:
Maybe:

```suggestion
// NOTE: Setting `maxAge` to `null` will instruct the browser to avoid deleting it -- leaving
// all session management up to the server and the Web application it serves. If it is
// not set to `null`, then either the cookie will always be removed after some fixed
// period of time even when the user is active -- otherwise, the `Set-Cookie` header
// will have to be sent on every request in order to advance `maxAge` to keep sessions
// open due to activity which creates significant overhead and more session
// management and synchronization complexity (between the server's session TTL and
// the client's cookie) that is usually not worth the cost.
```